### PR TITLE
Start security considerations

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

File: draft-uberti-avtcore-cryptex.md [hunk 1]
```diff
@@ -51,6 +51,25 @@ Encryption and Decryption
 Security Considerations
 =======================
 
+The specification does not change how SRTP authentication works in any
+way. It only changes encryption to expand the portion of the packet that
+is encrypted. Given theses two points, the authors do not feel it in any
```
Comment (juberti) [inline draft-uberti-avtcore-cryptex.md:56]: here, you can refer to the packet ciphertext section noted in my comment for #2

Comment (juberti) [inline draft-uberti-avtcore-cryptex.md:56]: I would make this more of a statement of fact than a feeling - given that we are encrypting more of the packet than before, this is necessarily an improvement.

File: draft-uberti-avtcore-cryptex.md [hunk 2]
```diff
@@ -51,6 +51,25 @@ Encryption and Decryption
...
+but not the number of identifiers.
+
+The leaves unencrypted the following items which are not encrypted by
+SRTP: the version of RTP, total length of extension, amount of padding,
+number of CSRC, the maker bit, the payload type number, the RTP sequence
+number, timestamp, and SSRC. These values are either need to be encrypt
+for SRTP processing or have very little information that has any value
+in encrypting. If any of theses values need to be encrypted, DTLS-SRTP
+is likely the wrong protocol to use and instead the RTP should be sent
+over a fully encrypted protocol such as DTLS or TLS.
```
Comment (juberti) [inline draft-uberti-avtcore-cryptex.md:71]: (which have additional per-packet overhead).

File: draft-uberti-avtcore-cryptex.md [hunk 3]
```diff
@@ -51,6 +51,25 @@ Encryption and Decryption
...
+way. It only changes encryption to expand the portion of the packet that
+is encrypted. Given theses two points, the authors do not feel it in any
+way makes the security worse than SRTP.
+
+This extension improves the security of SRTP by encrypting all the
+header extension data other than the total length of all the combined
+header extensions. It also encrypts the values of the CSRC identifiers
+but not the number of identifiers.
+
+The leaves unencrypted the following items which are not encrypted by
```
Comment (juberti) [inline draft-uberti-avtcore-cryptex.md:64]: I think we should have a separate section prior to security considerations on backwards compatibility which can talk about what we are not encrypting and why. You could probably reuse most of this text, and then refer to that briefly here in the security considerations.

**Comment by fluffy** [comment]
all the comments make sense - I will work on fixing up this PR

**Comment by juberti** [comment]
This text was incorporated into #13.

**Pull request closed without merging**